draft-gbb-ccamp-otn-path-computation-yang.md

```diff
@@ -180,11 +176,20 @@ sourcecode-markers="true" sourcecode-name="ietf-otn-path-computation@2022-07-10.
 
 # Manageability Considerations
 
-  TBD. 
+This document provides a method for requesting path computations for OTN tunnels. Consideration of mechanisms to gather and collate information required for the path computations will be necessary. Furthermore, storing path computation requests and responses and triggering actions according to configured will also need to be carefully managed and secured.
+
```

Comment:
.....according to configured will also need...
I would think the words missing would be "tunnels" probably but the sentence has to be completed

Reply:
Fixed in https://github.com/aguoietf/ietf-ccamp-otn-path-computation/pull/6/commits/8a51277379d170e062182dccb2bb388cd79cf49a